Documentation of getting-started in Japanese

tac0x2a opened the pull request:
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation

tac0x2a committed:
Remove machine_translated

tac0x2a committed:
Add 'For non-Linux operating systems and for AArch64 CPU' section

tac0x2a committed:
Translate getting-started/install.md to ja

tac0x2a committed:
Fix index.md anchors

tac0x2a committed:
Translate index.md via GitLocalize

tac0x2a committed:
Translate index.md via GitLocalize

tac0x2a committed:
Merge pull request #6 from tac0x2a/gitlocalize-12686

tac0x2a committed:
Translate getting-started/playground.md to ja

tac0x2a committed:
Merge branch 'documentation/ja' of github.com:tac0x2a/ClickHouse into documentation/ja

tac0x2a committed:
Translate getting-started/index.md to ja

tac0x2a committed:
Translate experiment to 試す in ja

tac0x2a committed:
Translate getting-started/tutoial.md to ja

tac0x2a committed:
Translate getting-started/tutoial.md to ja

tac0x2a committed:
Translate getting-started/example-datasets to ja

tac0x2a committed:
Fix original article path

tac0x2a committed:
Fix getting-started/index.md

blinkov commented on docs/ja/index.md:
```diff
@@ -1,66 +1,62 @@
-| \#N | …           | …          | …                  | …         | …                   |
+Row | WatchID | JavaEnable | Title | GoodEvent | EventTime
+--- | --- | --- | --- | --- | ---
+#0 | 89354350662 | 1 | Investor Relations | 1 | 2016-05-18 05:19:20
```
@tac0x2a could you revert these changes to tables? It seems like they are the reason for the failed tests.

tac0x2a replied:
@blinkov Thanks to review! I fixed it!

tac0x2a committed:
Revert  ja/index.md

tac0x2a committed:
Fix Input/Output to 入出力

tac0x2a committed:
Fix Original argicle en

blinkov commented:
@tac0x2a there are some broken links:
```

WARNING:mkdocs.structure.pages:Documentation file 'getting-started/playground.md' contains a link to 'getting-started/interfaces/jdbc.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/playground.md' contains a link to 'getting-started/interfaces/odbc.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/playground.md' contains a link to 'getting-started/(../interfaces/index.md)' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/tutorial.md' contains a link to 'sql-reference/statements/create/table.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/tutorial.md' contains a link to 'getting-started/interfaces/index.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/tutorial.md' contains a link to 'getting-started/sql-reference/statements/optimize.md' which is not found in the documentation files.
```

tac0x2a committed:
Fix missing links md files.

`getting-started/sql-reference/statements/optimize.md` is not placed in ja directory.
Currentry, it refer to english one.

tac0x2a committed:
Change link to optimize.md copied from en

blinkov commented:
@tac0x2a there's one more in the last commit: `WARNING:mkdocs.structure.pages:Documentation file 'sql-reference/statements/optimize.md' contains a link to 'sql-reference/statements/alter/index.md' which is not found in the documentation files.`

tac0x2a committed:
Change link to alter.md ja

blinkov commented:
@tac0x2a there're some broken links:
```
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/playground.md' contains a link to 'getting-started/interfaces/jdbc.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/playground.md' contains a link to 'getting-started/interfaces/odbc.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/playground.md' contains a link to 'getting-started/(../interfaces/index.md)' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/tutorial.md' contains a link to 'sql-reference/statements/create/table.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/tutorial.md' contains a link to 'getting-started/interfaces/index.md' which is not found in the documentation files.
WARNING:mkdocs.structure.pages:Documentation file 'getting-started/tutorial.md' contains a link to 'getting-started/sql-reference/statements/optimize.md' which is not found in the documentation files.
```

tac0x2a commented:
@blinkov Thank you for reporting. But, it looks same comment to [this](https://github.com/ClickHouse/ClickHouse/pull/13966#issuecomment-679861857). Is it collect ?

By the way, how can I check broken links on my PC ?

blinkov commented:
@tac0x2a yes, the last commit looks green, thanks!

For testing locally see: https://github.com/ClickHouse/ClickHouse/blob/master/docs/tools/README.md
